<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Usuarios</class>
 <widget class="QMainWindow" name="Usuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Usuarios</string>
  </property>
  <property name="windowIcon">
   <iconset resource="srsusergroup.qrc">
    <normaloff>:/user_group_users_accounts_contacts-128.png</normaloff>:/user_group_users_accounts_contacts-128.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>624</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ver Usuarios</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>661</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Saab</family>
           <pointsize>26</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ver Usuarios Registrados</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidgetPersonas">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ver Registros de Usuarios</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>471</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ver Registros de Usuarios</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Ingrese su ID:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">vercorreos.png</string>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>250</y>
        <width>411</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="lblId">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblnombre">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblDireccion">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblSexo">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCorreosE">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblCorreosR">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>181</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIRECCION</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>SEXO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CORREOS ENVIADOS</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CORREOS RECIBIDOS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="srsusergroup.qrc"/>
 </resources>
 <connections/>
</ui>
